Xamarin.Forms.Core/Easing.cs

```diff
@@ -75,6 +110,41 @@ static Easing()
 			});
 
 			BounceIn = new Easing(p => 1.0f - BounceOut.Ease(1 - p));
+
+			ExpoEaseOut = new Easing(ExponentialEquations.EaseOut);
```

Comment:
Is there any reason we can't initialize these at declaration (like the other Easing members)?
e.g. `public static readonly Easing ExpoEaseOut = new Easing(ExponentialEquations.EaseOut);`